email.html

```diff
@@ -1,7 +1,7 @@
 <!doctype html>
 <html>
   <head>
-    <meta name="viewport" content="width=device-width" />
+    <meta name="viewport" content="width=device-width" initial-scale=1.0/>
```

Comment:
`initial-scale` needs to be a comma separated value of the `content` attribute.

Reply:
needs to be `<meta name="viewport" content="width=device-width, initial-scale=1.0"/>

Reply:
Gotcha. @uttkarsh-cloud could you make this change? If you have a before and after screenshot as well I would love to understand what the visual difference is (I tested it myself and don't see anything, so wondering what I'm missing).